gatsby-config.js

```diff
@@ -1,6 +1,9 @@
 module.exports = {
   siteMetadata: {
     title: 'Common Knowledge',
+    description:
+      'Common Knowledge is a tech cooperative building tools and infrastructure to encourage grassroots political organising on the left.',
```

Comment:
@jamesmoulding is keen to use the term workers cooperative over tech cooperative.